### ISSUE TITLE
pkg init fails with /usr/bin/env: ‘ruby’: No such file or directory 

### ISSUE THREAD

**Issue by ddobrinskiy** [teaxyz/cli#382]
when trying to run `pkg init` for the first time, I get an error:

```
❯ pwd
/home/david/proj/pkgdev/pantry.extra

❯ pkg init                                                                                                                                    
/usr/bin/env: ‘ruby’: No such file or directory 
```

**Comment by ddobrinskiy** [comment]
Fixed by adding tea-packaged ruby bin to PATH:

`export PATH="$PATH:$HOME/.tea/ruby-lang.org/v3/bin"`

**Comment by mxcl** [comment]
This bug because I regressed knowing to add ruby to the env if the file has a `.rb` extension. Will fix. Thanks for the report.

**Comment by mxcl** [comment]
Fixed with brewkit ^0.3.
